espp2/datamodels.py

```diff
@@ -507,6 +507,8 @@ class Transactions(BaseModel):
 
     fromdate: date = None
     todate: date = None
+    opening_value_cash_usd: Optional[Decimal] = None
+    opening_value_csco_qty: Optional[Decimal] = None
```

Comment:
Could we make this independent of symbol you think?
Just in case we want to use Morgan atwork with !CSCO.
A dictionary keyed on symbol with Decimal as value?